packages/amplication-data-service-generator/src/server/message-broker/generate-message-broker-client-options/generate-message-broker-client-options.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  CreateMessageBrokerClientOptionsFactoryParams,
+  EventNames,
+  Module,
+} from "@amplication/code-gen-types";
+import pluginWrapper from "../../../plugin-wrapper";
+
+export async function createGenerateMessageBrokerClientOptionsFunction(
```

Comment:
why the duplicate naem "createGenerate" ?
```suggestion
export async function createMessageBrokerClientOptions(
```